python/res/enkf/queue_config.py

```diff
@@ -100,3 +116,16 @@ def lsf_server(self):
     @property
     def num_cpu(self):
         return self._get_num_cpu()
+
+    def __eq__(self, other):
```

Comment:
Please make sure that all data in the C struct is indeed compared. `num_cpu` is among others missing...

Reply:
All but the driver are tested for.